phys2bids/utils.py

```diff
@@ -95,7 +98,7 @@ def check_input_type(filename, indir):
             break
 
     if fftype_found:
-        print(f'File extension is .{ftype}')
+        LGR.info(f'File extension is .{ftype}')
```

Comment:
Here we could warn the users if they're looking for a particular extension txt) and we process another (acq).
Refers to #128 .